{"uuid":"9929d119-c6cb-4343-bedb-279966f4e638","historyId":"e12f91231a9e00e3601f8384c1d5bc45","fullName":"APi_Negative.Accounting.SG_NG_Analytics.AllForestWeatherDetails","labels":[{"name":"package","value":"APi_Negative.Accounting.SG_NG_Analytics"},{"name":"testClass","value":"APi_Negative.Accounting.SG_NG_Analytics"},{"name":"testMethod","value":"AllForestWeatherDetails"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"APi_Negative"},{"name":"subSuite","value":"APi_Negative.Accounting.SG_NG_Analytics"},{"name":"host","value":"DESKTOP-L0TTVHB"},{"name":"thread","value":"6732@DESKTOP-L0TTVHB.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"This story belongs to the All_Forest_Weather_Details"}],"links":[],"name":"AllForestWeatherDetails","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [400] but found [200]","trace":"java.lang.AssertionError: expected [400] but found [200]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1577)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat org.testng.Assert.assertEquals(Assert.java:665)\r\n\tat APi_Negative.Accounting.SG_NG_Analytics.AllForestWeatherDetails(SG_NG_Analytics.java:46)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Request Log","source":"4699409e-2815-4fab-baf5-11cb14b4def2-attachment","type":"application/json"},{"name":"Response Log","source":"0504b2e4-57a6-4117-a5e6-514696eaf096-attachment","type":"application/json"}],"parameters":[{"name":"arg2","value":"starts with zero"},{"name":"arg1","value":"400"},{"name":"arg0","value":"01"}],"start":1745482349084,"stop":1745482349268}